packages/palette/src/elements/TextArea/TextArea.tsx

```diff
@@ -193,6 +191,25 @@ const StyledTextArea = styled.textarea<StyledTextAreaProps>`
         cursor: default;
         ${TEXTAREA_STATES.disabled}
       }
+
+      ${props.title &&
+      css`
+        ::placeholder {
+          opacity: 0;
+        }
+      `}
     `
   }}
 `
+const StyledLabel = styled.label`
+  position: absolute;
+  top: 24px;
+  left: 6px;
+  padding: 0 5px;
+  background-color: ${themeGet("colors.white100")};
+  transform: translateY(-50%);
+  transition: 0.25s cubic-bezier(0.64, 0.05, 0.36, 1);
```

Comment:
Where does the specification for this transition come from?

Reply:
Ah, cool — yeah. Let's make sure we get some of this documented within some motion guidelines.